packages/templates/typescript/src/schema.handlebars

```diff
@@ -1,18 +1,20 @@
 {{#ifCond @root.config.resolvers "!==" false}}
+{{#if types}}
 import { GraphQLResolveInfo } from 'graphql';
 
-export type Resolver<Result, Args = any> = (
-  parent: any,
-  args: Args,
-  context: any,
-  info: GraphQLResolveInfo
+export type Resolver<Result, Parent = any, Context = any, Args = any> = (
+  parent?: Parent,
```

Comment:
hi, why is everything optional now? it is a bc break, all resolvers now have an incompatible signature

Reply:
Resolvers can work same without defining each parameter (parent, args, context, info). If you're using all of them, this shouldn't break your code.